keps/sig-network/20180612-ipv4-ipv6-dual-stack.md

```diff
@@ -333,36 +333,54 @@ or later. The dual-stack feature requires no changes to this API.
 The versions of CNI plugin binaries that must be used for proper dual-stack functionality (and IPv6 functionality in general) depend upon the version of the container runtime that is used in the cluster nodes (see [CNI issue #531](https://github.com/containernetworking/cni/issues/531) and [CNI plugins PR #113](https://github.com/containernetworking/plugins/pull/113)):
 
 ### Services
+Services depend on `--services-cluster-ip-range` for `ClusterIP` assignment. A dualstack cluster can have this flag configured as `--services-cluster-ip-rang=<cidr>,<cidr>` or `--services-cluster-ip-rang=<cidr>`. The following rules apply:
```

Comment:
Are we demanding that --cluster-cidr and --services-cluster-ip-range have the same ordering? How about node.spec.podCIDR ?

Reply:
`node.spec.podCIDR` stays the same before upgrade and after upgrade. `node.spec.podCIDRs` is assigned by node ipam controller and it ensures correct ordering according to the `–cluster-cidr` flag.

Reply:
and now we are not asking for the same order between `--cluster-cidr` and --services-`cluster-ip-range` one defines the default ip family for pods and one defines default ip family for service, they can be different (*but user can not change default during an upgrade*)

Reply:
OK, let's make sure that is clear for both flags.  In case of a single->dual upgrade they will necessarily be the same